build.gradle

```diff
@@ -96,4 +96,4 @@ detekt {
         config = "$projectDir/detekt.yml"
         filters = ".*/resources/.*,.*/tmp/.*"
     }
-}
\ No newline at end of file
+}
```

Comment:
Added a newline after this brace because it's our standard. Also because I creep.

Reply:
This is not creep. Missing this newline would/should cause a build failure due to static analysis.